### PR TITLE
wait for tooltips to show in CallComposite tests

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

adds wait on the tooltips aria-describedby label to show up before proceeding with the test
# Why

waits for the tool tip to appear since this is the new default state of the app when it runs since keyboard focus is no longer getting stuck in the modal.

# How Tested

ran tests locally and verified generated snapshots.

### CODE REVIEW

File: packages/react-composites/tests/browser/call/CallComposite.test.ts [hunk 1]
```diff
@@ -110,8 +110,8 @@ test.describe('Call Composite E2E CallPage Tests', () => {
 
       await page.goto(buildUrl(serverUrl, user));
       await waitForCallCompositeToLoad(page);
-      // Move the mouse off-screen to avoid tooltips / delayed focus from introducing flake in the snapshots
-      await page.mouse.move(0, 0);
+      // we need to dismiss any tool tips if there are any so we press the keyboard.
+      await page.keyboard.down('Escape');
```
Comment (prprabhu-ms) [inline packages/react-composites/tests/browser/call/CallComposite.test.ts:114]: Shouldn't this be https://playwright.dev/docs/api/class-keyboard#keyboard-press

Also, isn't this racy: The escape might be pressed before the tooltip shows, and then fail to dismiss the tooltip later?

File: packages/react-composites/tests/browser/call/CallComposite.test.ts [hunk 2]
```diff
@@ -110,8 +110,8 @@ test.describe('Call Composite E2E CallPage Tests', () => {
 
       await page.goto(buildUrl(serverUrl, user));
       await waitForCallCompositeToLoad(page);
-      // Move the mouse off-screen to avoid tooltips / delayed focus from introducing flake in the snapshots
-      await page.mouse.move(0, 0);
+      // we need to dismiss any tool tips if there are any so we press the keyboard.
```
Comment (prprabhu-ms) [inline packages/react-composites/tests/browser/call/CallComposite.test.ts:113]: Stale comment.

Comment (dmceachernmsft) [reply packages/react-composites/tests/browser/call/CallComposite.test.ts:113]: oops! will remove.

**Review by edwardlee-msft — APPROVED** [review]

**Comment by dmceachernmsft** [comment]
Closing this because unsure this fixes this. the end result is not needed anymore. will look into why the tooltips appeared in the release branch and not in main.

**Pull request closed without merging**